sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.IntegrationTests.WebApp/OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.WebApp.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <!-- 
+  Integration tests evaluate an app's components on a broader level than unit tests. 
+  Unit tests are used to test isolated software components, such as individual class methods. 
+  Integration tests confirm that two or more app components work together to produce an expected result, 
+  possibly including every component required to fully process a request.
+  https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-3.1
+  -->
+  
+  <PropertyGroup>
+    <RequiredTargetFrameworks>netcoreapp2.1</RequiredTargetFrameworks>
```

Comment:
Is this temporary, or the intention is test against multiple versions of runtime?

Reply:
temporary-ish.
The current frameworks include 4.6.1, which is not supported by the `Microsoft.AspNetCore.Mvc.Testing` framework.
This is also the reason why the pipeline is failing. I've reached out to the Azure SDK folks to get guidance on how to correctly handle this.

Reply:
resolved